core/kt-osrd-rjs-parser/src/main/kotlin/fr/sncf/osrd/RawInfraRJSParser.kt

```diff
@@ -801,7 +793,10 @@ fun parseRJSInfra(rjsInfra: RJSInfra): RawInfra {
                     props
                 )
             if (partId == null) {
-                // TODO warning
+                // TODO: raise warning
```

Comment:
Isn't the warning raised now ?

Reply:
To my understanding, the point of @eckter here is to add a warning to the response returned to editoast (or at least link the trace to a span of editoast) as mentionned here https://github.com/OpenRailAssociation/osrd/issues/6925

Reply:
Yeah, specifying would be perfect

Reply:
Done (amended) in https://github.com/OpenRailAssociation/osrd/pull/7792/commits/7d8c9e80d458828f1e0c23bd2404e56b070e4af3#diff-9325fe5faad772881e72d6024d73bade0606de7420d369192dd9486480914c48R796